deploy/list-connectors.ts

```diff
@@ -0,0 +1,75 @@
+import { glob } from "tinyglobby";
```

Comment:
suggestion:
you can use `mtx` extension so you can use top level await

Reply:
Um... I don't think we must use the top level await. so... once, just keep going the current state and considering it later.